getting-started.md

```diff
@@ -48,43 +46,51 @@ Once it is up and running you can get a shell inside the container with:
+
+### Create an application with AMS
+
+To create and manage an application from the command line, use AMS. 
+
+AMS requires a `manifest.yaml` file for your application. In its simplest form, the manifest looks like this:
 
 ```bash
-$ cat manifest.yaml
 name: com.my.game
 instance-type: a2.3
 ```
 
-> **Hint**: If you deployed Anbox Cloud with the streaming stack, use an instance type with GPU support like `g2.3`. Otherwise the container will not get access to a GPU for rendering and video encoding.
-
-The manifest basically defines the name of the application and which instance type the application should use as well as more advanced configuration like [Addons](https://discourse.ubuntu.com/t/managing-addons/17759), permissions and others . You can find more details about manifest format and the available instance types in the [Application Management](https://discourse.ubuntu.com/t/managing-applications/17760) and [Instance Types](https://discourse.ubuntu.com/t/instances-types-reference/17764) sections.
+It defines the name of the application and which instance type the application should use. The manifest can also contain more advanced configuration like [Addons](https://discourse.ubuntu.com/t/managing-addons/17759), permissions and others. You can find more details about the manifest format and the available instance types in the [Application Management](https://discourse.ubuntu.com/t/managing-applications/17760) and [Instance Types](https://discourse.ubuntu.com/t/instances-types-reference/17764) sections.
 
-To create the application with `ams`, place the APK as `app.apk` in the same directory as the `manifest.yaml`, and run the following command:
+To create the application with `ams`, place the APK of the Android application as `app.apk` in the same directory as the `manifest.yaml`. Then run the following command:
 
     amc application create /path/to/directory/with/apk/and/manifest/
 
-**AMS** will now run through a bootstrap process for the application to allow for faster boot times of the application later on.
+AMS will now run through a bootstrap process for the application to allow for faster boot times of the application later on.
 
 You can monitor the progress of the application with:
 
     watch -n 1 amc application ls
 
-When the application is marked as `ready`, you just have to publish it and it's ready to be used:
+When processing is done and the application is marked as `ready`, it is automatically published. Run the following command to check the publish state:
 
-    amc application publish <app name> 0
+    amc application show <app name>
 
-Now you can simply start a container for your new application by launching the application:
+If it says `published: true`, you can start a container for your new application by launching the application:
```

Comment:
On new installs, `auto-publish` will always be true, we can assume that all created applications will be published